### DISCUSSION TITLE
How to run on MacOS?

### DISCUSSION THREAD

**Discussion by zanderlewis** [preda/gpuowl#303] [Q&A]
How do you run PRPLL on MacOS? There are multiple errors after running the prpll binary after building, so I want to know if there is a way to run on a Mac.

(I have an Intel CPU if that helps)

**Comment by preda** [comment]
What GPU do you use on MacOS?
Do you have a working OpenCL install on your MacOS? -- one way to check is to install a program called "clinfo" or equivalent which displays some information about the OpenCL that is installed.

If all that works, GpuOwl should/might work.

In case it's not clear, GpuOwl works best on discrete GPUs ("big" GPUs that have good FP64). On integrated GPUs the performance is.. underwhelming due to a) poor memory bandwidth and b) poor FP64.
